src/test/moves/baton_pass.test.ts

```diff
@@ -74,13 +77,13 @@ describe("Moves - Baton Pass", () => {
 
     // round 2 - baton pass
     game.scene.getEnemyPokemon()!.hp = 100;
-    game.override.enemyMoveset(new Array(4).fill(Moves.BATON_PASS));
+    game.override.enemyMoveset([ Moves.BATON_PASS, Moves.BATON_PASS, Moves.BATON_PASS, Moves.BATON_PASS ]);
```

Comment:
```suggestion
    game.override.enemyMoveset(new Array(4).fill(Moves.BATON_PASS));
```
NIT: Reverting change